src/style.css

```diff
@@ -0,0 +1,84 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+body {
+  padding-top: 12px;
+}
+
+/* BIG SEARCH FIELD */
+.big-field {
+  width: 100%;
+  height: 70px;
+  margin-bottom: 20px;
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  padding-left: 62px;
+  outline: none;
+  position: relative;
+}
+
+.big-field:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.big-field:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+}
+
+.big-label {
+  position: absolute;
+  left: 40px;
+  top: 45px;
+}
+
+.big-label img {
+  width: 19px;
+  height: 19px;
+}
+
+/* SMALL SEARCH FIELD */
+.small-field {
+  width: 100%;
+  height: 42px;
+  font-size: 14px;
+  line-height: 19px;
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  color: #3d4e61;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  padding-left: 33px;
+  outline: none;
+  position: relative;
+}
+
+.small-field:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.small-field:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+}
+
+.small-label {
+  position: absolute;
+  left: 21px;
+  top: 122px;
+}
+
+.small-label img {
+  width: 11px;
+  height: 11px;
+}
```

Comment:
Don't repeat yourself. The position is the same for both images. You can create a common class for it